From 395c0f53ec226aaabedb166e6b3a7f8590b95a5f Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 11 May 2019 20:39:15 +0800
Subject: [PATCH] Make sure that Makefile doesn't try to compile these tests
 again on the target where the source dependencies won't be available.

Upstream-Status: Inappropriate [cross-compile specific]

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>

Rebase to 2.9.9
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 8f4e43d..5edb930 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -211,8 +211,7 @@ install-ptest:
 	sed -i -e 's|^Makefile:|_Makefile:|' $(DESTDIR)/Makefile
 	$(MAKE) -C python install-ptest
 
-runtests: runtest$(EXEEXT) testrecurse$(EXEEXT) testapi$(EXEEXT) \
-          testchar$(EXEEXT) testdict$(EXEEXT) runxmlconf$(EXEEXT)
+runtests:
 	[ -d test   ] || $(LN_S) $(srcdir)/test   .
 	[ -d result ] || $(LN_S) $(srcdir)/result .
 	$(CHECKER) ./runtest$(EXEEXT) && \
-- 
2.7.4

